WordPress/Classes/ViewRelated/Post/Prepublishing/PrepublishingViewController.swift

```diff
@@ -133,13 +133,11 @@ class PrepublishingViewController: UITableViewController {
     /// Toggles `keyboardShown` as the keyboard notifications come in
     private func configureKeyboardToggle() {
         NotificationCenter.default.publisher(for: UIResponder.keyboardDidShowNotification)
-            .map { _ in return true }
-            .assign(to: \.keyboardShown, on: self)
+            .sink { [weak self] _ in self?.keyboardShown = true }
```

Comment:
To make sure I understand it correctly: The cycle was due to passing directly `self` rather than using `assign` right?

Reply:
The `.assign(to: \.keyboardShown, on: self)` method retains `self`, which is a common [issue](https://forums.swift.org/t/does-assign-to-produce-memory-leaks/29546) in Combine.